instagram-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Overall solid work, just need to clean up code a bit (indentations ect).
Styling needs clean up as well.